sdks/wasm/src/driver.c

```diff
@@ -77,6 +81,8 @@ m_strdup (const char *str)
 
 static MonoDomain *root_domain;
 
+static int ZONE_INFO_FIRST_TIME = 1;
+
```

Comment:
Its better to have 0 as the default value, so it doesn't need to be initialized. Also, variable names should be lowercase.

Reply:
`gboolean`?
`char` for size?

Reply:
Yeah, not sure why I did 1 and not zero.

Reply:
This is no longer relevant with current implementation as the flag was removed